.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,65 @@
+{
+  "bcd": {
+    "scope": "json",
+    "prefix": ["bcd", "browser-compat"],
+    "body": [
+      "{",
+      "  \"${1|api,css,html,http,javascript,mathml,svg,webdriver,webextensions,xpath,xslt|}\": {",
+      "    \"${2:${TM_FILENAME_BASE}}\": {",
```

Comment:
This makes it insert the filename with no extension (e.g. in `AbortController.json` this inserts just `AbortController`) and then marks it as a placeholder that the user can modify afterward.

Reply:
Very cool!